bot/translation/translation.py

```diff
@@ -0,0 +1,183 @@
+"""Contains a Cog for all funcionality regarding translations."""
+
+import discord
+from discord.ext import commands
+
+from bot import constants as const
+from bot.logger import command_log, log
+from bot.persistence import DatabaseConnector
+
+
+class TranslationCog(commands.Cog):
+    """Cog for translation functions."""
+
+    def __init__(self, bot):
+        """Initializes the Cog.
+
+        Args:
+            bot (discord.ext.commands.Bot): The bot for which this cog should be enabled.
+        """
+        self.bot = bot
+        self._db_connector = DatabaseConnector(const.DB_FILE_PATH, const.DB_INIT_SCRIPT)
+
+        # Role instances
+        self.role_moderator = bot.get_guild(int(const.SERVER_ID)).get_role(int(const.ROLE_ID_MODERATOR))
+
+    # A special method that registers as a commands.check() for every command and subcommand in this cog.
+    async def cog_check(self, ctx):
+        return self.role_moderator in ctx.author.roles
+
+    @commands.group(name='translation', hidden=True, invoke_without_command=True)
+    @command_log
+    async def translation(self, ctx: commands.Context):
+        await ctx.send_help(ctx.command)
+
+    @translation.command(name='add')
+    @command_log
+    async def add_translation(self, ctx: commands.Context, message: discord.Message, flag_emoji: str,
```

Comment:
If you want to restrict which emojis are allowed (eg only flags), you could create a hardcoded list.

Reply:
But then the list would contain somewhere around 250 entries. And I don't really want to add only a few flags because then we would limit ourself which languages can be added.

Reply:
We should still make a hardcoded llist with at least a few flags. I mean it can easily be extended.